src/PiggyCustomEnchants/EventListener.php

```diff
@@ -1022,40 +1022,9 @@ public function checkArmorEnchants(Entity $entity, Event $event)
                     if ($enchantment !== null) {
                         if ($event->getDamage() >= $entity->getHealth()) {
                             if ($enchantment->getLevel() > 1) {
-                                switch($slot){
-                                    case ArmorInventory::SLOT_CHEST:
-                                        $entity->getArmorInventory()->setChestplate($this->plugin->addEnchantment($armor, $enchantment->getId(), $enchantment->getLevel() - 1));
-                                        break;
-                                    case ArmorInventory::SLOT_FEET:
-                                        $entity->getArmorInventory()->setBoots($this->plugin->addEnchantment($armor, $enchantment->getId(), $enchantment->getLevel() - 1));
-                                        break;
-                                    case ArmorInventory::SLOT_HEAD:
-                                        $entity->getArmorInventory()->setHelmet($this->plugin->addEnchantment($armor, $enchantment->getId(), $enchantment->getLevel() - 1));
-                                        break;
-                                    case ArmorInventory::SLOT_LEGS:
-                                        $entity->getArmorInventory()->setChestplate($this->plugin->addEnchantment($armor, $enchantment->getId(), $enchantment->getLevel() - 1));
-                                        break;
-                                    default:
-                                        $this->plugin->getLogger()->error("CheckArmorEnchants: Slot $slot not found in ArmorInventory");
-                                }
-
+                                $entity->getArmorInventory()->setItem($slot,$this->plugin->addEnchantment($armor, $enchantment->getId(), $enchantment->getLevel() - 1));
```

Comment:
Formatting issue. Missing space here but not below...

Reply:
Updated in:
https://github.com/DaPigGuy/PiggyCustomEnchants/pull/122/commits/39e6a03f4c47fb1045d3889bcce4b56ede511607